project/hackyourweather/src/pages/CityChart.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+import { useParams, useHistory } from "react-router-dom";
+import {
+  XAxis,
+  YAxis,
+  Tooltip,
+  CartesianGrid,
+  Area,
+  AreaChart,
+} from "recharts";
+
+const CityChart = () => {
+  const [city, SetCity] = useState({});
+  const [cityIsFound, setCityIsFound] = useState(true);
+
+  const params = useParams();
+  const history = useHistory();
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const response = await fetch(
+          `http://api.openweathermap.org/data/2.5/forecast?id=${params.cityId}&appid=${process.env.REACT_APP_OPENWEATHERMAP_API_KEY}`
+        );
+        const requestedCity = await response.json();
+        requestedCity.message ? setCityIsFound(false) : setCityIsFound(true);
+        const cityName = `${requestedCity.city.name}, ${requestedCity.city.country}`;
+        const chartData = requestedCity.list.map(
+          (item) =>
+            (item = {
```

Comment:
I don't see any need for the reassignment over here.
```suggestion
            ({
```

please let me know if I missed something you thought of.

Reply:
Yes, there is no need actually to the reassignment